data/packages/com.unity.uiextensions.yml

```diff
@@ -2,6 +2,7 @@ name: com.unity.uiextensions
 displayName: Unity UI Extensions
 description: The Unity UI Extensions project is a collection of extension scripts / effects and controls to enhance your Unity UI experience. These scripts have been gathered from many sources, combined and improved over time.
 repoUrl: 'https://github.com/Unity-UI-Extensions/com.unity.uiextensions'
+readme: 'upm:Packages/com.unity.uiextensions/README.md'
```

Comment:
```
readme: 'upm:Packages/com.unity.uiextensions/README.md'
```
`upm` - is the branch
`Packages/com.unity.uiextensions/README.md` is the path of the README.md file

In this case, it should be 

```
readme: 'release:README.md'
```

Reply:
righteo, will get that updated asap, thanks